src/exports.jl

```diff
@@ -1114,7 +1114,7 @@ export minimal_faces
 export minimal_generating_set
 export minimal_generator
 export minimal_size_generating_set, has_minimal_size_generating_set, set_minimal_size_generating_set
-export minimal_supercone_coordinates, minimal_supercone_indices
+export minimal_supercone_coordinates, minimal_supercone_indices, is_minimal_supercone_coordinate_vector, standard_coordinates
```

Comment:
please put these into four separate lines (and sort alphabetically)

Reply:
For more background: @fingolfin complained a while back that it was hard to find exports. In the case at hand, one might be looking for `export standard_coordinates` and would never find it. The agreement was to collect all exports from `Oscar/src` in `src/exports` and order all export alphabetically.

(Exports from experimental "modules" have their own `exports.jl` file. Again, the desire is to have one line per export and have them ordered alphabetically.)

Reply:
It is explicitly mentioned here https://docs.oscar-system.org/dev/DeveloperDocumentation/styleguide/#Code-structure

Reply:
although, the documention mentions putting things in one line in certain cases, which applies to the situation here

Reply:
also the line above puts multiple things in one line

Reply:
I would argue that in the case here, we have more or less different functions. In the case of the line above, all three functions there are intrinsically connected to one another, as they are all defined via one common call to `@gapattribute`

Reply:
Thanks